<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="widget">
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>60</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>打开图片</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>560</width>
      <height>360</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>400</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="currentTabCB">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>170</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>多次处理</string>
    </property>
   </widget>
   <widget class="QListWidget" name="algorithmList">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>160</y>
      <width>131</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openFolderButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>选择图集</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>播放/暂停</string>
    </property>
   </widget>
   <widget class="QSlider" name="playSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="serialPortButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口助手</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
